src/datamigration/dags/datamigration_utils/hive_bq_load_utils.py

```diff
@@ -160,7 +160,9 @@ def get_table_size(tbl, table_gs_path):
         cmd, capture_output=True, shell=True, encoding="utf-8"
     )
     res = truncate_result.stdout.split(" ")
+    print(len(res))
```

Comment:
Seems like statement added just for testing

Reply:
Deleted